docs/migrations.md

```diff
@@ -0,0 +1,18 @@
+# Migration guides
+
+## From v9 to v10
+
+1. New dynamic values support.
+   In v9, observables and function values have been part of JSS core. In v10 they are standalone plugins. If you are using the default preset you don't need to do anything else than upgrading the default preset and JSS. If you are using a custom setup and you use function values or observables, you will have to add 2 additional plugins. See [plugins.md](./plugins.md).
+
+1. Full syntax support in dynamic values.
```

Comment:
2.

Reply:
haha, no, this is how markdown works, look at the preview

Reply:
WTF